bot-twitter/src/variables.ts

```diff
@@ -0,0 +1,2 @@
+export const ETHEREUM_NETWORK = process.env.ETHEREUM_NETWORK || 'kovan';
+export const KEEPER_MIN_PROFIT_DAI = process.env.KEEPER_MIN_PROFIT_DAI || 100;
```

Comment:
`process.env.KEEPER_MIN_PROFIT_DAI` is a string or undefined. You need to parse it before using it. Let's remove default value to validate that it works and let's skip keeper if the variable is not set.

Also, please make sure that the new variables are added into the dockerfile and everywhere else it needs to be added to work in production

Reply:
Don't I just have to add the values to the secrets?

Reply:
> Don't I just have to add the values to the secrets?

If you're not sure, building the image local and running it with appropriate env variables is a proper way to answer this question 😅

Reply:
I checked the Dockerfiles. In the frontend we are baking them in. However in the Bot Twitter we are not. Seeing as we are already using enviroment variables for the Twitter connection, I _assume_ they get backed in via the camber sections of the DockerFile. But I will test this theory.